client/signup/steps/site-type/form.jsx

```diff
@@ -66,7 +68,15 @@ class SiteTypeForm extends Component {
 }
 
 export default connect(
-	null,
+	state => {
+		const siteId = getSelectedSiteId( state );
+
+		return {
+			// TODO: Better handling for Jetpack flows in the site-type lib,
+			// so we don't depend on injecting conditionals into components like this.
+			notJetpack: ! isJetpackSite( state, siteId ),
+		};
+	},
```

Comment:
I'd tend towards a positive identifier here, i.e., `isJetpack`. 

`notJetpack` leaves open the question, _if not Jetpack, then what_?

If the intent behind the condition is "everything but Jetpack" then it would still make sense to write:

` ! this.props.isJetpack && siteTypeProperties.purchaseRequired...`

You could simplify the syntax a little too. Totally optional.

```
	state => (  {
			// TODO: Better handling for Jetpack flows in the site-type lib,
			// so we don't depend on injecting conditionals into components like this.
			isJetpack: isJetpackSite( state, getSelectedSiteId( state ) ),
		} )
```

Reply:
I borrowed `notJetpack` from `jetpack-only.js`... but I agree `isJetpack` is better. Will update, thanks!